tests/test_instruments_qblox_cluster_qcm_bb.py

```diff
@@ -38,65 +32,59 @@
 O4_OFFSET = 0.5890
 O4_QUBIT = 4
 
+connected_controller
+controller
```

Comment:
?

Reply:
I've added this very ugly lines since pycln automatically remove the import of controller and connected_controller. If removed they will give an error since cluster fixture depends on controller fixture.

Reply:
Ok I managed to find a comment (`#noqa`) in order to avoid that pycln remove the unused (but necessary) imports.